deliver/lib/deliver/upload_metadata.rb

```diff
@@ -111,6 +111,7 @@ def upload(options)
       end
 
       v.release_on_approval = options[:automatic_release]
+      v.toggle_phased_release(enabled: !!options[:phased_release])
```

Comment:
> you ship a new release using fastlane deliver with the default value?

I tried in that situation, it works well. 👍 
This value might be ignored when we ship a new release.